com.unity.render-pipelines.high-definition/Editor/Lighting/HDLightUI.cs

```diff
@@ -240,7 +240,8 @@ static void DrawShapeContent(SerializedHDLight serialized, Editor owner)
 
                 case HDLightType.Point:
                     EditorGUI.BeginChangeCheck();
-                    EditorGUILayout.PropertyField(serialized.shapeRadius, s_Styles.lightRadius);
+                    using (new EditorGUI.DisabledScope(false))
```

Comment:
A disableScope with (false) do nothing. The value is to know if this should be disable or not.

Reply:
that code was not supposed to be submitted, my apologies, will remove those changes (locally i was testing disabled(true) versus hiding fields)